.github/workflows/ci.yml

```diff
@@ -29,3 +32,15 @@ jobs:
       - run: pnpm lint
       - run: pnpm build
       - run: pnpm vitest ${{ matrix.testenv.args }}
+      - run: pnpm benchmarks
```

Comment:
how long does this take to run?

Reply:
I wonder if we should move this to a separate job, so we don't need to bump the timeout here, and can have CI status report sooner for the tests

Reply:
`pnpm benchmarks` seems to take around 3min
I agree it is better to move it to a separate job. I have updated the PR.